### DISCUSSION TITLE
c:geo 2024 in numbers - merry Christmas & a happy new year!

### DISCUSSION THREAD

**Discussion by moving-bits** [cgeo/cgeo#16457] [Show and tell]
## 2024 has been a busy year for c:geo

Here are some numbers:

- **1614 tickets** on support (c:geo only, without WhereYouGo)
- **926 commits** on GitHub (77 of those coming from bots - dependabot and translations). This year, all contributions came from c:geo team directly.
- **17 releases** in 2024 (2+1 major/feature releases + 14 minor/bugfix releases) - "2+1" because one feature release needed a second try on Play Store due to the changed conditions about background services. Some of the releases had release candidates additionally.

Play Store lists an installation base of **660k users** (average value of last 30 days).
- 35% D, 11% USA, 10% CZ, 6% F, 4% NL
- 8% Android 15, 48% Android 14, 16% Android 13, 9% Android 12 (and ~4600 users still on Android 5/6, 5400 Android 7, 9800 Android 8)
- **72000 reviews** in total / average **rating 4.5***
- Current vitals are 0,45% for crashes / 0,71% for ANR - most of those are coming from the old maps (but crash rate is currently increasing slightly due to #16430)

Some of the larger new features in 2024 were
- Integrated Wherigo player
- UnifiedMap
- Edit logs
- Elevation chart for routes / tracks
- Grouping of cache lists
- Manual mitigation for Captcha login problems
- ...

as well as many more smaller optimizations and bug fixes - you'll find the details in our [combined release notes](https://github.com/cgeo/cgeo/releases)

**Quite a year, I'd like to say!**

**Thanks to the c:geo team** for being such a wonderful group, for everybody contributing her/his expertise, for great discussions in a positive tone, devoting their spare time for c:geo, always striving to make c:geo even better - it's a pleasure working in such a community!

And last but not least **thanks to all contributors** for helping with c:geo being such a great geocaching app it is, be it by opening a GitHub issue, code or translation contributions, a forum discussion or just explaining some functionality to a fellow geocacher - thanks to all of you!

**Comment by Lineflyer** [comment]
Sounds like all fantastic numbers!!

Thanks to all of you...I wish I have had more time for c:geo in 2024 (both in using and helping here). Maybe next year its getting a bit more quiet with other topics.
